webui/react/src/components/Link.tsx

```diff
@@ -36,10 +30,21 @@ const Link: React.FC<Props> = ({
     } else {
       routeAll(url);
     }
-  }, [ onClick, path, popout ]);
+  };
+};
+
+const Link: React.FC<Props> = ({
+  disabled, inherit, path, popout, onClick, children,
+}: PropsWithChildren<Props>) => {
+  const classes = [ css.base ];
+  const rel = windowFeatures.join(' ');
+
+  if (!disabled) classes.push(css.link);
+  if (inherit) classes.push(css.inherit);
 
```

Comment:
suggestion: using a `useCallback` to avoid regenerating the click handler on each call.
```
const handleLinkClick = useCallback(() => handleClick(path, onClick, popout), [ path, onClick, popout ]);
```

Reply:
would `useMemo` work better here?

Reply:
`useCallback` is basically `useMemo` for specifically for an end result of a function, which is the case here.

Reply:
the correct line is:
```
const handleLinkClick = useCallback(handleClick(path, onClick, popout), [ path, onClick, popout ]);
```

Reply:
I updated it